EIPS/eip-7907.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: 7907
+title: Remove Contract Code Size Limit
+description: Removes the contract code size limit introduced in EIP-170 and adds a gas metering to code loading
+author: Charles Cooper (@charles-cooper)
```

Comment:
```suggestion
author: Charles Cooper (@charles-cooper), Qi Zhou (@qizhou)
```

Since this draws heavily from Qi Zhou's work I think he should be credited.


Qi Zhou ([@qizhou](https://github.com/qizhou))

Reply:
It's basically the same EIP with different constants.

Reply:
I am glad to see the progress in increasing contract code size, which is a long pain for contract developers. Feel free to add me as a co-author; I would love to push the EIP further together.

Reply:
18bb9c10e3de068568c11d70fa0648ff6a6075e5